test/unit_test/unit_test.vcxproj

```diff
@@ -35,7 +35,7 @@
     <ClCompile Include="error_test.cc" />
     <ClCompile Include="example_header_test.cc" />
     <ClCompile Include="explore_test.cc" />
-    <ClCompile Include="flatbuffer_parser_test.cc" />
+    <ClCompile Include="flatbuffer_parser_test.cc" Condition="Exists('$(VcpkgIntegration)')" />
```

Comment:
Is it possible to condition this on `BUILD_FLATBUFFERS` or some env var?

Reply:
I tried it out